.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
41 files....it's 2:41am and I have to look through 41 files to find your work...

if your project structure is correct, then you need to work on your .gitignores. Include any ide specific file paths like this one (.idea) so I don't have to look through them all. Also any .bin folders can go too.

Reply:
project structure wasn't right...gitignore ended up a layer below this...which is why it didn't pick it up